docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -351,7 +353,7 @@ Release
 
    .. prompt:: bash [vagrant@localhost]$
 
-      echo https://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box
+      echo https://s3.amazonaws.com/clusterhq-archive/vagrant/tutorial/flocker-tutorial-${VERSION}.box
```

Comment:
Remove the section above, starting "# Copy the tutorial box to the final location"